src/main/java/Task.java

```diff
@@ -0,0 +1,127 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Task {
+    //Attributes
+    protected TaskType taskType;
+    protected String name;
+    protected boolean isDone;
+    private static ArrayList<Task> tasks = new ArrayList<Task>();
+
+    //Constructors
+    Task (String name) {
+        this.name = name;
+        isDone = false;
+        taskType = TaskType.TASK;
+    }
+
+    public TaskType getTaskType () {
+        return taskType;
+    }
+
+    //Methods
+    public String toString () {
+        return (isDone ? "[X] " : "[ ] ") + name;
+    }
+
+    public static void listTasks () {
+        System.out.println(UI.LINE_SEPARATOR);
+        for (int i = 0; i < tasks.size(); i++) {
+            System.out.print((i + 1) + ".");
+            System.out.println(tasks.get(i));
+        }
+        System.out.println(UI.LINE_SEPARATOR);
+    }
+
+    public static void addTask (Task task) {
+        System.out.println(UI.LINE_SEPARATOR);
+
+        switch (task.getTaskType()) {
+            case DEADLINE:
+            case EVENT:
+            case TODO:
+                System.out.println("Got it. I've added this task:");
+                tasks.add(task);
+                break;
+        }
```

Comment:
Do follow the coding standard for switch block on our module website, i.e. ```switch``` and ```case``` should be aligned, there's no extra indentation in front of ```case```.